### ISSUE TITLE
Add alt text to test status icons on Ecosystem page

### ISSUE THREAD

**Issue by eddybrando** [Qiskit/qiskit.org#2653]
### Feature description

The [ecosystem page](https://qiskit.org/ecosystem/) has cards that have tables with information about the test status of the packages. This information is given in the form of coloured icons:

<img width="675" alt="Screenshot 2022-06-07 at 17 41 53" src="https://user-images.githubusercontent.com/22047320/172423352-0028979d-28d4-4ae3-a5aa-11323c0e08bf.png">
 
We need to add an alt text to make the icons accessible by providing a description of the icons.

These are the alt texts that should be added:

- Passing tests (for the green icon)
- Failing tests (for the red icon)
- No test information (for the gray icon)

### Additional context

https://www.w3.org/WAI/tutorials/images/informative/

**Comment by rutterjt** [comment]
Hey there, mind if I take this one?

**Comment by javabster** [comment]
Go for it @rutterjt! Remember to read the CONTRIBUTING.md and let us know if you have any questions 😄
